Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ProblemsController.cs

```diff
@@ -710,5 +735,61 @@ private void RetestSubmission(int submissionId)
             entry.Property(pr => pr.Processed).IsModified = true;
             entry.Property(pr => pr.Processing).IsModified = true;
         }
+
+        private DetailedProblemViewModel PrepareProblemViewModelForEdit(int id)
+        {
+            return this.Data.Problems.All()
+                .Where(x => x.Id == id)
+                .Select(DetailedProblemViewModel.FromProblem)
+                .FirstOrDefault();
+        }
+
+        private DetailedProblemViewModel PrepareProblemViewModelForCreate(Contest contest)
+        {
+            var lastOrderBy = -1;
```

Comment:
Maybe this must be extracted as a constant. If I don't know the logic, I won't really understand this number. It's a  bit "magic"